src/js/components/projects/createProjectModal/summary.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import GitHubUserAvatar from '@/js/components/githubUsers/avatar';
+import UserCards from '@/js/components/githubUsers/cards';
+import { ModalCard } from '@/js/components/tasks/capture';
+import { Dependency } from '@/js/store/projects/reducer';
+import { GitHubOrg, GitHubUser } from '@/js/store/user/reducer';
+
+interface Props {
+  name: string;
+  selectedOrg?: GitHubOrg;
+  selectedCollaborators: GitHubUser[];
+  selectedDeps: Dependency[];
+}
+
+const CreateProjectSummary = ({
+  name,
+  selectedOrg,
+  selectedCollaborators,
+  selectedDeps,
+}: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <form className="slds-form slds-p-around_large">
+      <h3 className="slds-text-heading_small slds-m-bottom_medium">
```

Comment:
this component seems to have a lot of H3s. Is this a valid heading structure? It is important that headers follow a logical hierarchy

Reply:
Hm, my understanding (from e.g. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)) is that multiple H3s is a valid structure for repeated headings of a similar level.

Since SLDS [updated the Modal heading to an H1](https://github.com/salesforce/design-system-react/releases/tag/v0.10.48), I think these should probably be H2s though -- I'll update that (and other similar sub-headings inside Modals).

Adjusted in https://github.com/SFDO-Tooling/Metecho/pull/2019/commits/8b50b4fc5c793962be86b31a7706446e63cc8a49